### DISCUSSION TITLE
How should i configure vim for Haskell?

### DISCUSSION THREAD

**Discussion by oxinabox** [google-research/dex-lang#408] [Q&A]
Does anyone have sensible (neo-)vim settings for Haskell that make it follow the conventions that are in the DexLang source code?

It defaults to tabs instead of spaces.
When I try and look up how to configure vim for haskell I end up seeing things tellimg me to install like 4 plugins,
which seem excesssive.
Is there some minimal set of settings with just one or two plugins, that make it do the right thing?
Maybe I just need to configure some indenting rule like `ts=2; expandtabs`?

**Comment by apaszke** [comment]
I'm using a very barebones vim setup, and only have a bunch of those language-generic formatting settings specified in my `.vimrc`. The full source is [here](https://github.com/apaszke/dotfiles/blob/master/.vimrc) if you're interested.
